src/v/cluster/tx_gateway_frontend.cc

```diff
@@ -1112,7 +1112,7 @@ ss::future<add_paritions_tx_reply> tx_gateway_frontend::do_add_partition_to_tx(
         break;
     }
 
-    auto status = co_await stm->add_partitions(tx.id, partitions);
+    auto status = co_await stm->add_partitions(term, tx.id, partitions);
```

Comment:
(similar question as before)

am I missing something, this comment seems to break term change improvements we did by comparing current term with tx.etag?

Reply:
when `get_tx` detects a transferring tx it updates its etag so it should be up to date